joblib/logger.py

```diff
@@ -19,6 +19,9 @@
 
 from .disk import mkdirp
 
+logger = logging.getLogger("joblib")
+logger.addHandler(logging.NullHandler())
```

Comment:
Why putting a `NullHandler`? This changes the behavior as the messages are not printed anymore. Using a `StreamHandler` makes more sense IMO.

Reply:
The general advice given for [logging in a library](https://docs.python.org/3.6/howto/logging.html#configuring-logging-for-a-library) is to use a `NullHandler` so as not to interfere with an application's logging setup. However, I'm fine with adding a `StreamHandler` by default in this case since it's easy enough to remove it when it's not added to the root logger.

Reply:
I think it is better to add the `StreamHandler` as these message will only appear if `Memory.verbose >= 1`.

Reply:
Shouldn't that be the user's decision? One can always add/modify a handler to a named logger if necessary (I often modify the log level of the `werkzeug` logger as I find it too verbose for my needs).